### PR TITLE
chore(http): Remove range request example no longer returning 'accept-ranges: none'

### LINKED ISSUE #16073: Need an example site that actually sends 'Accept-Ranges: none' in responses
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests

### What specific section or headline is this issue about?

curl -I https://www.youtube.com/watch?v=EwTZ2xpQwpA

### What information was incorrect, unhelpful, or incomplete?

Look... 1 curled URL directs to a complex Html page, which has corresponding non-media type 2 and as such has no Accept-Ranges: none 3 and surely not in such transcription 4 because youtube uses all-lowercase for headers. Be careful with easy solutions.

### What did you expect to see?

Something which does not cross an idea of the article

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

I love you guys in a different ways

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/http/range_requests`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/http/range_requests/index.md
* Last commit: https://github.com/mdn/content/commit/0872b73731bb333479087e069e0ba94009587fb7
* Document last modified: 2022-04-22T06:07:10.000Z

</details>

### PR DESCRIPTION
**Pull request by bsmth**
### Description

The example used to return `Accept-Ranges: none`, but this is no longer the case.
Removing the example completely and doing some tidy while I'm at it.

### Related issues and pull requests

Fixes #16073

### CODE REVIEW

File: files/en-us/web/http/range_requests/index.md [hunk 1]
```diff
@@ -6,70 +6,83 @@ page-type: guide
 
 {{HTTPSidebar}}
 
-An HTTP {{HTTPHeader("Range")}} request asks the server to send only a portion of an HTTP message back to a client. Range requests are useful for clients like media players that support random access, data tools that know they need only part of a large file, and download managers that let the user pause and resume the download.
+An HTTP {{HTTPHeader("Range")}} request asks the server to send parts of a resource back to a client.
+Range requests are useful for clients like media players that support random access, data tools that know they need only part of a large file, and download managers that let the user pause and resume a download.
```
Comment (dipikabh) [inline files/en-us/web/http/range_requests/index.md:10]: nit to improve clarity and to remove "know"
```suggestion
Range requests are useful for various clients, including media players that support random access, data tools that require only part of a large file, and download managers that let users pause and resume a download.
```

File: files/en-us/web/http/range_requests/index.md [hunk 2]
```diff
@@ -6,70 +6,83 @@ page-type: guide
...
 
 ## Checking if a server supports partial requests
 
-If an HTTP response includes the {{HTTPHeader("Accept-Ranges")}} header and its value is anything other than `none`, then the server supports range requests. You can perform a manual check by issuing a {{HTTPMethod("HEAD")}} request with a tool like cURL.
+If an HTTP response includes the {{HTTPHeader("Accept-Ranges")}} header with any value other than `none`, the server supports range requests.
+If responses omit the `Accept-Ranges` header, it indicates the server doesn't support partial requests.
+If range requests are not supported, applications can adapt to this condition; for instance, download managers can disable pause buttons that relied on range requests to resume a download.
+
+To check if a server supports range requests, you can issue a {{HTTPMethod("HEAD")}} request so that you can inspect headers without requesting the resource in full.
```
Comment (dipikabh) [inline files/en-us/web/http/range_requests/index.md:18]:
```suggestion
To check if a server supports range requests, you can issue a {{HTTPMethod("HEAD")}} request to inspect headers without requesting the resource in full.
```

File: files/en-us/web/http/range_requests/index.md [hunk 3]
```diff
@@ -6,70 +6,83 @@ page-type: guide
...
+content-length: 1024
+content-range: bytes 0-1023/146515
 …
+
 (binary content)
 ```
 
-The {{HTTPHeader("Content-Length")}} header now indicates the size of the requested range (and not the full size of the image). The {{HTTPHeader("Content-Range")}} response header indicates where in the full resource this partial message belongs.
+The {{HTTPHeader("Content-Length")}} header indicates the size of the requested range (and not the full size of the image).
+The {{HTTPHeader("Content-Range")}} response header indicates where in the full resource this partial message belongs.
```
Comment (dipikabh) [inline files/en-us/web/http/range_requests/index.md:85]:
```suggestion
The {{HTTPHeader("Content-Length")}} header indicates the size of the requested range, not the full size of the image.
The {{HTTPHeader("Content-Range")}} response header indicates where this partial message belongs within the full resource.
```

File: files/en-us/web/http/range_requests/index.md [hunk 4]
```diff
@@ -79,7 +92,9 @@ The {{HTTPHeader("Range")}} header also allows you to get multiple ranges at onc
 curl http://www.example.com -i -H "Range: bytes=0-50, 100-150"
 ```
 
-The server responses with the {{HTTPStatus("206")}} `Partial Content` status and a {{HTTPHeader("Content-Type")}}`: multipart/byteranges; boundary=3d6b6a416f9b5` header, indicating that a multipart byterange follows. Each part contains its own `Content-Type` and `Content-Range` fields and the required boundary parameter specifies the boundary string used to separate each body-part.
+The server responds with the {{HTTPStatus("206", "206 Partial Content")}} status as follows.
```
Comment (dipikabh) [inline files/en-us/web/http/range_requests/index.md:95]:
```suggestion
The server responds with the {{HTTPStatus("206", "206 Partial Content")}} status as shown below.
```

File: files/en-us/web/http/range_requests/index.md [hunk 5]
```diff
@@ -79,7 +92,9 @@ The {{HTTPHeader("Range")}} header also allows you to get multiple ranges at onc
...
+The response contains a {{HTTPHeader("Content-Type")}} header, indicating that a multipart byterange follows.
+The boundary string (`3d6b6a416f9b5` in this case) separates each body part, which has its own `Content-Type` and `Content-Range` fields:
```
Comment (dipikabh) [inline files/en-us/web/http/range_requests/index.md:97]:
```suggestion
The boundary string (`3d6b6a416f9b5` in this case) separates the body parts, each of which has its own `Content-Type` and `Content-Range` fields:
```

**Review by dipikabh — APPROVED** [review]
LGTM @bsmth, thanks!

Leaving a +1 so you can merge after making the changes.

**Comment by bsmth** [comment]
Super, thank you @dipikabh 👍🏻

File: files/en-us/web/http/range_requests/index.md [hunk 6]
```diff
@@ -6,70 +6,83 @@ page-type: guide
...
 {{HTTPSidebar}}
 
-An HTTP {{HTTPHeader("Range")}} request asks the server to send only a portion of an HTTP message back to a client. Range requests are useful for clients like media players that support random access, data tools that know they need only part of a large file, and download managers that let the user pause and resume the download.
+An HTTP {{HTTPHeader("Range")}} request asks the server to send parts of a resource back to a client.
+Range requests are useful for various clients, including media players that support random access, data tools that require only part of a large file, and download managers that let users pause and resume a download.
 
 ## Checking if a server supports partial requests
 
-If an HTTP response includes the {{HTTPHeader("Accept-Ranges")}} header and its value is anything other than `none`, then the server supports range requests. You can perform a manual check by issuing a {{HTTPMethod("HEAD")}} request with a tool like cURL.
+If an HTTP response includes the {{HTTPHeader("Accept-Ranges")}} header with any value other than `none`, the server supports range requests.
```
Comment (bsmth) [inline files/en-us/web/http/range_requests/index.md:14]: Side note: some older browsers relied on `Accept-Ranges: none` response header to disable the 'pause' and 'resume' buttons in the download manager. Nowadays:

>If responses omit the `Accept-Ranges` header, it indicates the server doesn't support partial requests.

**Pull request merged**